src/index.js

```diff
@@ -71,3 +71,12 @@ exports.marshalPrivateKey = (key, type) => {
 
   return key.bytes
 }
+
+exports.randomBytes = (number) => {
```

Comment:
11:39 <@daviddias> that " needs to be exposed" is difference
11:39 <@daviddias> because what I really wanted is to give a length and get a buffer
11:39 <@daviddias> I updated my PR to use that function
11:39 <@daviddias> but it is just adding a op